package.json

```diff
@@ -1,14 +1,15 @@
 {
   "name": "memoize-fs",
   "version": "2.0.0",
-  "description": "memoize/cache in file system solution for Node.js",
+  "description": "Node.js solution for memoizing/caching a function and its return state into the file system",
   "author": "Boris Diakur <contact@borisdiakur.com> (https://github.com/borisdiakur)",
   "scripts": {
-    "test": "npm run eslint && mocha test",
-    "eslint": "eslint .",
-    "mocha": "mocha --reporter nyan test",
-    "istanbul": "rm -rf coverage && ./node_modules/.bin/istanbul cover ./node_modules/mocha/bin/_mocha --report html && open coverage/memoize-fs/index.js.html",
-    "coveralls": "rm -rf coverage && ./node_modules/.bin/istanbul cover ./node_modules/mocha/bin/_mocha --report lcovonly -- -R spec && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js && rm -rf ./coverage"
+    "lint": "eslint .",
+    "pretest": "npm run lint",
+    "test": "nyc mocha",
+    "mocha": "mocha -R nyan",
+    "coverage": "nyc report | coveralls",
```

Comment:
One last favor: Can you change this line to `"coverage": "nyc report --reporter=text-lcov | coveralls",` ?

Reply:
Ah, maybe good catch.

Reply:
We have already set `text` and `lcov` reporters in the `nyc` config defined later in package.josn.

The `lcov` reporter outputs both `lcovonly` (same as `text-lcov`) and html.

Reply:
Without `--reporter=text-lcov` I’m getting the following error:

```
[error] "2020-02-28T16:28:48.924Z"  'error from lcovParse: ' 'Failed to parse string'
[error] "2020-02-28T16:28:48.927Z"  'input: ' '----------|---------|----------|---------|---------|-------------------\n' +
  'File      | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s \n' +
  '----------|---------|----------|---------|---------|-------------------\n' +
  'All files |     100 |      100 |     100 |     100 |                   \n' +
  ' index.js |     100 |      100 |     100 |     100 |                   \n' +
  '----------|---------|----------|---------|---------|-------------------\n'
[error] "2020-02-28T16:28:48.927Z"  'error from convertLcovToCoveralls'

/Users/m267843/workspace/open_source/memoize-fs/node_modules/coveralls/bin/coveralls.js:19
      throw err;
      ^
Failed to parse string
(Use `node --trace-uncaught ...` to show where the exception was thrown)
```